azure-pipelines.yml

```diff
@@ -95,7 +95,7 @@ jobs:
           set -e
           GOARCH=amd64 GOOS=linux go build -o azcopy_linux_amd64
           export AZCOPY_E2E_EXECUTABLE_PATH=$(pwd)/azcopy_linux_amd64
-          go test -timeout 20m -race -short -cover ./e2etest
```

Comment:
Changes in this file will be discarded.

Reply:
+1. As long as we don't update the 60min timeout.